main.go

```diff
@@ -443,3 +452,20 @@ This means that when the API changes
 	log.Donef("Success")
 	log.Printf("The app (.ipa) was successfully uploaded to [iTunes Connect](https://itunesconnect.apple.com), you should see it in the *Prerelease* section on the app's iTunes Connect page!")
 }
+
+func copyToTmp(pth string) (string, error) {
```

Comment:
the purpose of this function is to fix / normalise the ipa path, i would name it like: fixArtifactPath / normalizeArtifactPath